shim.php

```diff
@@ -10,6 +10,44 @@ function toXPath($cssExpr, $prefix = 'descendant-or-self::') {
     }
 }
 
+// Add aliases for PHPUnit 6
+
+namespace {
+    if (version_compare(PHP_VERSION, '7.0.0') >= 0 && class_exists('PHPUnit\Framework\TestCase')) {
```

Comment:
I found some weird behaviour when running the tests on PHP 5.6, which installs PHPUnit 5.7.*, I think. When I tried with the condition `class_exists('PHPUnit\Framework\TestCase')` it was returning true, which is incorrect, because this class is only present in PHPUnit 6, which can only be installed in PHP 7. I think it's because the class can be autoloaded. It's probably safe to check for PHP's version only.